### ISSUE TITLE
Problem with PIDFILE

### ISSUE THREAD

**Issue by aanriot** [dalibo/temboard#186]
Hello,

When $PIDDIR does not exist, the _temboard/debian/temboard.init_ script exits in _/lib/lsb/init-functions.d/40-systemd_, after  the sourcing of _/lib/lsb/init-functions_.

The _temboard_ systemd unit returns the following error:

```
mai 12 14:22:24 server temboard[2358]: FATAL: can't write pidfile /var/run/temboard/temboard.pid
```
It appears on Debian Jessie (amd64) with the latest packages available.

I quickly checked without figuring out why it happens.

Thanks.

**Comment by bersace** [comment]
Yes, init script needs some love. Let's make sure it's production ready for 1.0.

**Comment by bersace** [comment]
Fixed by f54c164bf57ae7f61fc90358952731853735b4e9
